config/prow/cluster/cerberus_configmaps.yaml

```diff
@@ -62,15 +62,15 @@ data:
         logging.info("Check health of prow controller\n")
 
     def check():
-        pod_name =  runcommand.invoke("kubectl -n prow get pods -o custom-columns=\":metadata.name\" | grep controller")
+        pod_name =  runcommand.invoke("kubectl -n prow get pods -o custom-columns=\":metadata.name\" | grep prow-controller-manager")
         pod_log = runcommand.invoke("kubectl -n prow logs %s" % (pod_name))
```

Comment:
you can use instead:
```
kubectl logs deploy/prow-controller-manager -n prow
```

Reply:
But the cerberus Service account needs a change of permissions to use this:
Now I see
```
Error from server (Forbidden): deployments.apps "prow-controller-manager" is forbidden: User "system:serviceaccount:prow:cerberus" cannot get resource "deployments" in API group "apps" in the namespace "prow"
2022-01-03 13:16:07,042 [ERROR] Failed to run kubectl logs deploy/prow-controller-manager -n prow
```

Reply:
@mkumatag

Reply:
you can add that in the rbac here: https://github.com/ppc64le-cloud/test-infra/blob/83c90522c506b655bf52a96bc25ec7517f6db529/config/prow/cluster/cerberus_rbac.yaml#L16